docs/testing.md

```diff
@@ -40,6 +40,10 @@ To run just a subset of them, specify a particular file or directory:
     py.test tests/unit
     py.test tests/unit/api.py
 
+To run just a pep8 test:
+
+    py.test --pep8 -m pep8
```

Comment:
@zenhack @henn I changed it to * already.  For the record though I tested it and it did not run against .venv either